drivers/wifi/nrf700x/zephyr/inc/zephyr_fmac_main.h

```diff
@@ -78,6 +78,13 @@ struct wifi_nrf_ctx_zep {
 	struct rpu_conf_params conf_params;
 #endif /* CONFIG_NRF700X_WIFI_UTIL */
 #endif /* CONFIG_NRF700X_RADIO_TEST */
+	u8 *extended_capa, *extended_capa_mask;
+	unsigned int extended_capa_len;
+	struct wifi_nrf_ext_capa {
+		enum nrf_wifi_iftype iftype;
+		u8 *ext_capa, *ext_capa_mask;
+		unsigned int ext_capa_len;
+	} iface_ext_capa[NRF_WIFI_IFTYPE_MAX];
```

Comment:
Why not just move this to `struct wifi_nrf_vif_ctx_zep` ?

Reply:
Since this is device/radio level capabilities.

Reply:
Ok...but these are still per interface so it would make sense to have them in the interface specific structure.

Reply:
Ah ok you are referring to iface_ext_capa specifically. I only remember the radio specific params. Doesn't make much of a difference but ok, we can move those ..